cobalt/build/configs/linux-x64x11-evergreen/args.gn

```diff
@@ -0,0 +1,3 @@
+import("//cobalt/build/configs/linux-x64x11/args.gn")
+
+use_custom_libc = true
```

Comment:
there's already a use_custom_libcxx , made a similar flag to get musl changes in